### PR TITLE
Updates icon sizes to be the same size/type as Steam

### PR DESCRIPTION
**Pull request by vreelb**
This should make it really easy to contribute the icon step of a new game. Size-wise the jpg and png files differed based on the image being stored/compressed, some were bigger and some were smaller but they averaged to be about the same size.

* Updates existing icons to those from Steam for Steam games, manually made remaining icons from other sources.
* Updates `README.md` to reflect these changes.
* Updates the step order to have creating the folder first, then simple steps, then the `yaml` at the end.
* CSS changes to support both the new size and also smaller viewscreens:
image

### CODE REVIEW

**Review by AsherGlick — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ ResourceCalculator.com is a website that can calculate required resources from a
 Creating or Updating a Calculator
 =================================
 If you want to add a new resource list to the calculator, you will only need 4 things:  
-**1)** The list of resources and what is needed to obtain them (eg: [resource_lists/minecraft/resources.yaml](resource_lists/minecraft/resources.yaml))  
-**2)** A set of rectangular images of uniform size for each of the resources (eg: [resource_lists/minecraft/items/](resource_lists/minecraft/items/))  
-**3)** A 300px by 150px thumbnail of the game you are making a calculator for (eg: [resource_lists/minecraft/icon.png](resource_lists/minecraft/icon.png))  
-**4)** A folder in `resource_lists/` for all the files to live in (eg: [resource_lists/minecraft](resource_lists/minecraft))  
+1. A folder in `resource_lists/` for all the files to live in (eg: [resource_lists/minecraft](resource_lists/minecraft))
+2. A **460px** by **215px** thumbnail of the game you are making a calculator for (eg: [resource_lists/minecraft/icon.jpg](resource_lists/minecraft/icon.jpg))
+    * These are the same dimensions as the thumbnail in the Steam store.
```
[the remark this answers is not in the document]
Comment (vreelb) [reply README.md:11]: Yeah I think that would be helpful. I also want to include a note that we should aim for pictures that aren't temporary for a sale / event and are more general to the game,

[the remark this answers is not in the document]
Comment (vreelb) [reply README.md:11]: That Deep Rock example is actually very helpful as it shows that there have been 13 headers so far and you can decrement the number to get earlier ones.

File: core/index.html [hunk 2]
```diff
@@ -19,9 +19,19 @@
 
 
 		{% for calculator in calculators %}
-			<a href="./{{calculator.path}}"><div class="calculator_link_wrapper" style="background-image: url('{{calculator.path}}/icon.png')"><div class="calculator_link" >{{calculator.display_name}}</div></div></a>
+			<a href="./{{calculator.path}}">
```
[the remark this answers is not in the document]
Comment (vreelb) [reply core/index.html:22]: Was it a problem when building? I didn't run into any issues when testing this.

[the remark this answers is not in the document]
Comment (AsherGlick) [reply core/index.html:22]: Closing this after offline discussion that this wont effect production but might effect folks using a draft build.

**Pull request merged**